tutor/src/models/task-plans/teacher/plan.js

```diff
@@ -302,7 +302,6 @@ class TeacherTaskPlan extends BaseModel {
     return new TeacherTaskPlan({
       ...this.clonedAttributes,
       tasking_plans: course.periods.active.map(period => ({
-        opens_at: course.starts_at,
```

Comment:
Need to test to make sure assignments created on the day a course starts still work, even if the course has a weird starts_at time, like 10PM.

Reply:
yeah, I removed that because the tasking was overriding it so having it here didn't make sense.. https://github.com/openstax/tutor-js/blob/master/tutor/src/models/task-plans/teacher/tasking.js#L29-L33

Reply:
Hmm if a course starts on a day after the default open time and you try to create an assignment on that day, will that blow up?

Reply:
yep, I think it would.. I'll update that code. thanks!